servant/src/Servant/API/Raw.hs

```diff
@@ -11,4 +12,5 @@ import Data.Typeable (Typeable)
 -- In addition to just letting you plug in your existing WAI 'Application's,
 -- this can also be used with 'Servant.Utils.StaticFiles.serveDirectory' to serve
 -- static files stored in a particular directory on your filesystem
-data Raw deriving Typeable
+newtype Raw a (m :: * -> *) = Raw a
```

Comment:
[Tagged](http://hackage.haskell.org/package/tagged-0.7.2/docs/Data-Tagged.html) may have a few more instances that we should also add.